localization/english/culture/wc_cultures_l_english.yml

```
@@ -0,0 +1,402 @@
+ frostborn:0 "Frostborn"
+ frostborn_collective_noun:1 "Frostborn"
+
+ #Elven group
+ elf_group:0 "Elven"
+ elf_group_collective_noun:1 "Elves"
+ high_elf:0 "High Elf"
+ high_elf_collective_noun:1 "High Elves"
+ blood_elf:0 "Blood Elf"
+ blood_elf_collective_noun:1 "Blood Elves"
+ night_elf:0 "Night Elf"
+ night_elf_collective_noun:1 "Night Elves"
+ highborne:0 "Highborne"
+ highborne_collective_noun:1 "Highborne"
+ nightborne:0 "Nightborne"
+ nightborne_collective_noun:1 "Nightborne"
+ naga:0 "Naga"
+ naga_collective_noun:1 "Naga"
+ satyr:0 "Satyr"
+ satyr_collective_noun:1 "Satyrs"
+
+ #Troll group
+ troll_group:0 "Troll"
+ troll_group_collective_noun:1 "Trolls"
+ gurubashi:0 "Gurubashi"
+ gurubashi_collective_noun:1 "Gurubashi"
+ amani:0 "Amani"
+ amani_collective_noun:1 "Amani"
+ drakkari:0 "Drakkari"
+ drakkari_collective_noun:1 "Drakkari"
+ farraki:0 "Farraki"
+ farraki_collective_noun:1 "Farraki"
+ zandalari:0 "Zandalari"
+ zandalari_collective_noun:1 "Zandalari"
+ dark_troll:0 "Dark Troll"
+ dark_troll_collective_noun:1 "Dark Trolls"
+ blood_troll:0 "Nazmani"
+ blood_troll_collective_noun:1 "Nazmani"
+
+ #Tauren group
+ tauren_group:0 "Yaungol"
+ tauren_group_collective_noun:1 "Yaungol"
+ taunka:0 "Taunka"
+ taunka_collective_noun:1 "Taunka"
+ tauren:0 "Tauren"
+ tauren_collective_noun:1 "Tauren"
+ highmountain:0 "Higmountain"
+ highmountain_collective_noun:1 "Highmountains"
+ yaungol:0 "Yaungol"
+ yaungol_collective_noun:1 "Yaungol"
+
+ #Vrykul group
+ vrykul_group:0 "Vrykul"
+ vrykul_group_collective_noun:1 "Vrykul"
+ vrykul:0 "Vrykul"
+ vrykul_collective_noun:1 "Vrykul"
+ kvaldir:0 "Kvaldir"
+ kvaldir_collective_noun:1 "Kvaldir"
+ hyldnir:0 "Hyldnir"
+ hyldnir_collective_noun:1 "Hyldnir"
+ frost_vrykul:0 "Frost Vrykul"
+ frost_vrykul_collective_noun:1 "Frost Vrykul"
+ drust:0 "Drust"
+ drust_collective_noun:1 "Drust"
+
+ #Aqir group
+ aqir_group:0 "Aqir"
+ aqir_group_collective_noun:1 "Aqir"
+ qiraji:0 "Qiraji"
+ qiraji_collective_noun:1 "Qiraji"
+ nerubian:0 "Nerubian"
+ nerubian_collective_noun:1 "Nerubians"
+ mantid:0 "Mantid"
+ mantid_collective_noun:1 "Mantid"
+
+ #Dragon group
+ dragon_group:0 "Dragon"
+ dragon_group_collective_noun:1 "Dragons"
+ green_dragon:0 "Green Dragon"
+ green_dragon_collective_noun:1 "Green Dragons"
+ red_dragon:0 "Red Dragon"
+ red_dragon_collective_noun:1 "Red Dragons"
+ blue_dragon:0 "Blue Dragon"
+ blue_dragon_collective_noun:1 "Blue Dragons"
+ black_dragon:0 "Black Dragon"
+ black_dragon_collective_noun:1 "Black Dragons"
+ bronze_dragon:0 "Bronze Dragon"
+ bronze_dragon_collective_noun:1 "Bronze Dragons"
+
+ #Undead group
+ undead_group:0 "Undead"
+ undead_group_collective_noun:1 "Undead"
+ scourge:0 "Scourge"
+ scourge_collective_noun:1 "Scourged"
+ forsaken:0 "Forsaken"
+ forsaken_collective_noun:1 "Forsaken"
+
+ #Ancient group
+ ancient_group:0 "Ancient"
+ ancient_group_collective_noun:1 "Ancients"
+ protector:0 "Protector"
+ protector_collective_noun:1 "Protectors"
```

It's their main culture.
```suggestion
 protector:0 "Ancient"
 protector_collective_noun:1 "Ancients"
```